apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/JobCreationOptions.cs

```diff
@@ -59,5 +59,18 @@ public abstract class JobCreationOptions
         /// Label keys must start with a letter.
         /// </summary>
         public IDictionary<string, string> Labels { get; set; }
+
+        /// <summary>
+        /// The ID of the session in which to create the job.
+        /// If specified it is Used to resolve _SESSION appearing as the dataset id.
```

Comment:
We definitely shouldn't have a capital U - but I don't understand this sentence anyway, to be honest.

Perhaps:

"If specified, this is used to associates the job with a given session"

(That's adapting wording from https://cloud.google.com/bigquery/docs/reference/rest/v2/ConnectionProperty)

Reply:
done